### PR TITLE
data : SohamRoy-01

### PR DESCRIPTION
**Pull request by SohamRoy-01**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4977"></a>

### CODE REVIEW

File: data/SohamRoy-01.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+
+  "name": "Soham Roy",
+  "type": "personal",
+  "bio": "Frontend Developer passionate about full stack and web3 technologies",
+  "avatar":"https://github.com/SohamRoy-01.png",
```
Comment (Hrishi5111998) [inline data/SohamRoy-01.json:6]: hey @SohamRoy-01, you don't need an avatar so can remove it. thanks for contributing!

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]

File: data/SohamRoy-01.json (hunk 1 above)
Comment (SohamRoy-01) [reply data/SohamRoy-01.json]: @Hrishi5111998 Ok I removed it....

**Review by Sindhuinti — APPROVED** [review]
LG 👍🏻

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉

**Pull request merged**